docs/tech_leadership_principles_guidance.md

```diff
@@ -0,0 +1,68 @@
+
+Once escalated, two TOC members will be assigned to guide and mentor the project or group and its leadership on methods and options to arrive at an agreeable resolution. Resolutions are crafted to allow for the best outcome for the project or group and its future. The TOC may suggest changes among the leadership roles, responsibilities, and project governance to support that outcome.
+
+## Additional Information and Guidance
+
+### Conflict is likely
+
+Innovation is often born from the diversity of opinion, perspective, and experience of individuals collaborating towards a common goal or objective. Fostering an engaging and exploratory discussion, debate, and exchange of ideas to drive innovation requires a foundation of respect, grace, and curiosity so all voices may be elevated, heard, and incorporated into the discussion.
+
+Any personal, social views and political opinions exposed to the community can destroy and undermine trust of the leader holding them because those personal views may not align with CNCF values and may be seen to promote intolerance. Individuals have a right to their personal, private perspectives, but as leaders in our community we are expected to embody and practice the technical leadership principles and uphold our Code of Conduct through our behavior and engagements, serving as the arbiter of equity and fairness within projects, within the community, within CNCF spaces, and at CNCF events.
+
+### Building and breaking trust
+
+Leadership creates a power imbalance and in our community this imbalance is offset through an individuals’ ability to positively influence contributors to collaborate towards agreed outcomes. Our nomination and voting process for filling leadership roles in our groups, projects, or committees) is rooted in the merits of an individual’s work and the trust built as a result of their behaviors and community engagement. It is that trust that allows for positive influence to occur to offset any negative harm that could occur due to such an imbalance of power.
+
+When an individual's behavior fails to embody our principles and values and/or their personal/private views are exposed to the community, it has the potential to subversively abuse that imbalance and negatively harm the project and its community. A leader’s behavior and views has the singular ability to build or break our projects and our community.
+
+### Prevention
+
+All projects, groups, and committees are expected to establish a policy or governance process that allows for the removal of a leader for failure to continually align with our principles and values through a vote of no confidence after attempts to constructively correct (restorative justice) have occurred. It is unrealistic to detail every possible offense or behavior that could warrant such action and each project may have additional considerations to evaluate as part of determining the need for a vote of no confidence. 
```

Comment:
All projects, groups and committees are expected to establish …

Was looking for a resource, found this one: https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md#involuntary-removal-or-demotion

May be good to link it. And Or mention tag contrib strat governance reviews for help.